### PR TITLE
About page styling

### LINKED ISSUE #281: About page: styling
Please update the About page to match the styling we have in the Figma design.

We're going to need to add links for everyone.

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Fixes #281

## How Has This Been Tested?

locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] User visible change (users will notice UI or functional changes)

## Previewing Changes

1. Go to https://s4us-pr-747.onrender.com/about.

## Checklist:

- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by edwingl10** [comment]
missing background image in the repo

Screen Shot 2021-11-29 at 10 55 51 PM

File: src/img/img6.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 26.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!-- Generator: Adobe Illustrator 23.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (josueetcom) [inline src/img/img6.svg:2]: How come this file shows up in the diff?

File: src/pages/About.jsx [hunk 2]
```diff
@@ -117,16 +124,28 @@ function About() {
...
-        Looking for scholarships?
-      </Typography>
-      <Button
-        component={Link}
-        to="/scholarships"
-        variant="contained"
-        color="primary">
-        Browse Scholarships
-      </Button>
+      <Grid
```
Comment (josueetcom) [inline src/pages/About.jsx:127]: I'm guessing this component is similar to the one in UserHome.jsx right? We should probably refactor this into its own component file and reuse it across both pages.

**Review by josueetcom — COMMENTED** [review]
My only concern is why there are SVG file changes.

File: src/img/img6.svg (hunk 1 above)
Comment (edwingl10) [reply src/img/img6.svg:2]: I just resized it to better fit the container it was in. 
This made is easier to work with when putting it in the top section.

File: src/pages/About.jsx (hunk 2 above)
Comment (edwingl10) [reply src/pages/About.jsx:127]: yes this is the part I mentioned in the other PR.

File: src/components/GridContent.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import img5 from '../img/img5.svg';
+import { Link } from 'react-router-dom';
+import { Grid, Paper, Typography, Button } from '@mui/material';
+
+function GridContent() {
```
Comment (edwingl10) [inline src/components/GridContent.jsx:6]: I wasn't sure what to call this.

Comment (josueetcom) [reply src/components/GridContent.jsx:6]: Hmmm, how about `LookingForScholarshipsBanner`?

File: src/components/GridContent.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <Grid
+      container
+      component={Paper}
+      variant="outlined"
+      sx={{ padding: { xs: 2, md: 3 }, mt: 3 }}>
```
Comment (josueetcom) [inline src/components/GridContent.jsx:12]: The `mt` looks a bit extraneous. Isn't going to affect the UserHome page as well?

**Review by josueetcom — APPROVED** [review]

File: src/components/GridContent.jsx (hunk 4 above)
Comment (edwingl10) [reply src/components/GridContent.jsx]: I'll add the margin in the other page.

**Pull request merged**